### PR TITLE
wheatleyinabox/components

### PR DESCRIPTION
**Pull request by wheatleyinabox**
# PR Checklist

- Briefly describe what changes you are making and the Issue they are addressing
Implements Tabs and Nav Tabs as components from #5
- [x] Did you provide proper documentation for your new changes?
N/A
- [x] Did you add test cases for your new changes?
N/A
- [x] Are there any new environment variables / deployment changes that are needed? If so, check this box after notifying an admin.
N/A

### CODE REVIEW

File: src/components/Tab.tsx [hunk 1]
```diff
@@ -0,0 +1,180 @@
+import TabSvg from "./svgs/TabSvg";
+
+// Property Set Up //
+
+// Orientation
+export enum orientation {
```
Comment (blbergo) [inline src/components/Tab.tsx:6]: capitalize because its a constant

File: src/components/Tab.tsx [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+  horizontal = "flex-row p-3",
```
Comment (blbergo) [inline src/components/Tab.tsx:7]: same here

File: src/components/Tab.tsx [hunk 3]
```diff
@@ -0,0 +1,180 @@
...
+        <p className="m-0">{text}</p>
+      </button>
+    </div>
+  );
+}
+
+export default Tab;
+
+/*
+can be deleted later!
```
Comment (blbergo) [inline src/components/Tab.tsx:154]: this is awesome btw

File: src/components/svgs/TabSvg.tsx
Comment (blbergo) [inline src/components/svgs/TabSvg.tsx:1]: this can just be just an `<Image>` component that returns an SVG from a `.svg file`

**Review by blbergo — APPROVED** [review]
very good, just address my 2 comments and you are all set :)

File: src/components/svgs/TabSvg.tsx
Comment (wheatleyinabox) [reply src/components/svgs/TabSvg.tsx:1]: Alright I made another commit to change it to this! 

However, I'm still figuring out how I can pass the fill color to the `.svg file`. Working on it now!

**Pull request merged**